devices/danfoss.js

```diff
@@ -78,7 +79,18 @@ module.exports = [
                 .withDescription('Mean radiator load for room calculated by gateway for load balancing purposes (-8000=undefined)')
                 .withValueMin(-8000).withValueMax(2000),
             exposes.numeric('load_estimate', ea.STATE_GET)
-                .withDescription('Load estimate on this radiator')],
+                .withDescription('Load estimate on this radiator'),
+            exposes.binary('preheat_status', ea.STATE_GET, true, false)
+                .withDescription('Specific for pre-heat running in Zigbee Weekly Schedule mode'),
+            exposes.enum('adaptation_run_status', ea.STATE_GET, ['None', 'In progress', 'Found', 'Lost'])
+                .withDescription('bit0=Adaptation run in progress, bit1=Valve Characteristic found, bit2=Valve Characteristic lost'),
```

Comment:
Don't think we should talk about bits at this level, also fix the enum values e.g. (`in_progress`)